CHANGELOG.md

```diff
@@ -1,8 +1,20 @@
-# Change log
+# Changelog
 
 All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
-These should not impact the functionality of the module.
+These should not affect the functionality of the module.
+
+## [v2.1.0](https://github.com/voxpupuli/puppet-dhcp/tree/v2.1.0) (2017-09-25)
+[Full Changelog](https://github.com/voxpupuli/puppet-dhcp/compare/v2.0.0...v2.1.0)
+
+**Implemented enhancements:**
+
+- Update $extra\_config to work with arrays and not only strings [\#156](https://github.com/voxpupuli/puppet-dhcp/pull/156) ([bastelfreak](https://github.com/bastelfreak))
+- Support SmartOS [\#155](https://github.com/voxpupuli/puppet-dhcp/pull/155) ([petems](https://github.com/petems))
+
+**Merged pull requests:**
+
+- release 2.0.1 [\#153](https://github.com/voxpupuli/puppet-dhcp/pull/153) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
This tag was never pushed to github and the forge?

Reply:
oh :(
That got lost somehow

Reply:
Can we rectify this beforre merging this PR?  I assume we need an admin to cut a release manually to the forge.

Reply:
I don't see which feature was actually added. Maybe the PR should be renamed to 'Prepare 2.0.1'.

Reply:
@ekohl IMO we should release this as 2.1.0, we added two new features. I'm not sure if 2.0.1 was supposed to be released, but I don't think so.

Reply:
@bastelfreak that's what I meant. If we renamed the #153 to ```Prepare 2.0.1``` then the changelog matches reality and all would be well.

Reply:
ah got you, will do